### PR TITLE
Task 10

### PR DESCRIPTION
**Pull request by TanyaLevchenya**

### CODE REVIEW

File: task_10/Rabbit.css [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+}
+.name-section{
+     font-size: 35px;
+     height: 40px;
+     line-height: 40px;
+     padding: 5px 10px;
+     margin-left: 100px;
+     text-align: center;  
+}
+ div h3:first-child{
```
Comment (Killerkip10) [inline task_10/Rabbit.css:100]: Please don't this this selector anymore)

.name-section h3:first-child - it is more correct

File: task_10/Rabbit.css [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+}
+/* section */
+
+.main{
+    margin-left: 100px;
+    margin-right: 100px;
+}
+
+/* First conteiner-text  */
+.conteiner{
```
Comment (Killerkip10) [inline task_10/Rabbit.css:116]: .container

File: task_10/Rabbit.css [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+}
+.image{
+    height: 240px;
+    width: 240px;
+    border-radius: 120px;
+    border: 6px solid #ff9c00;
+    margin: 15px 0;
+}
+span{
+    color: #fb9c06;
```
Comment (Killerkip10) [inline task_10/Rabbit.css:167]: You set color for all span's in the page?)))

Please don't do it anymore

File: task_10/Rabiit.html [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+</section>
+<!-- section -->
+
+<main>
+    <div class="main">
+
+    <!-- First conteiner-text -->
+        <div class="conteiner">
+
+            <div class="block">
```
Comment (Killerkip10) [inline task_10/Rabiit.html:81]: Not understandable class name

**Review by Killerkip10 — COMMENTED** [review]
## Task 10
### Total 100 points

File: task_10/Rabbit.css (hunk 2 above)
Comment (TanyaLevchenya) [reply task_10/Rabbit.css:116]: )))), вот он на лицо английскиййййййй

File: task_10/Rabbit.css (hunk 3 above)
Comment (TanyaLevchenya) [reply task_10/Rabbit.css:167]: хорошо

**Pull request merged**